docgen/src/Styling.md

```diff
@@ -1,9 +1,7 @@
 ---
 title: Styling
 layout: guide.pug
-nav_groups:
-  - core
-nav_sort: 1
+category: core
```

Comment:
So category: core means it's a guide? Maybe we could express this by using category: guide